### PR TITLE
Added Materialize 'Chips' to tags

### LINKED ISSUE #18: Cat tags should be individual 'tags'
Not styled like this, but the same idea

image

### LINKED ISSUE #19: Delete Root Favicon folder

### PR DESCRIPTION
**Pull request by Bouncey**
closes #18 
closes #19

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -11,6 +11,9 @@ var count = 0; // Keeping track of number of images
 
 photosRef.once("value")
   .then(function(snapshot) {
+
+    console.log(snapshot);
```
Comment (Bouncey) [inline src/js/main.js:15]: This is stripped out during `uglify` and doesn't make it to `static/main.js`

**Comment by Bouncey** [comment]
image

File: src/index.html [hunk 2]
```diff
@@ -53,7 +53,7 @@
 					</div>
 					<div class="row">
 						<div class="input-field col s6">
-							<input placeholder="Enter Tags" id="tags" type="text" required>
+							<input placeholder="Speparate tags with a comma" id="tags" type="text" required>
```
Comment (ghost) [inline src/index.html]: ~~Speparate~~ Separate

Comment (Bouncey) [inline src/index.html]: Here we go hahahahaha :laughing:

Comment (Bouncey) [inline src/index.html]: Let me know when you are done

Comment (ghost) [inline src/index.html]: @Bouncey :joy: that's it :wink:

**Pull request merged**